ding/reward_model/gail_irl_model.py

```diff
@@ -201,6 +133,7 @@ def load_expert_data(self) -> None:
         with open(self.cfg.data_path + '/expert_data.pkl', 'rb') as f:
             self.expert_data_loader: list = pickle.load(f)
         self.expert_data = self.concat_state_action_pairs(self.expert_data_loader)
+        self.expert_data = torch.unbind(self.expert_data, dim=0)
```

Comment:
why unbind here

Reply:
because we re-use the concat_state_action_pair function, and its return is different from the original function in Gail. So I used unbind here.